scripts/travis/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+if [[ $TASKS == "clang-format" ]]; then
+  cd $TRAVIS_BUILD_DIR
+  ./scripts/travis/run_clang_format_diff.sh master $TRAVIS_COMMIT
+else
+  # First, get the super module dir
+  cd ..
+  git clone https://github.com/openchemistry/openchemistry
+  cd openchemistry
+  git submodule init avogadroapp avogadrodata molequeue
+  git submodule update
+
+  # Move the trial avogadrolibs into the open chemistry dir
+  mv ../avogadrolibs .
+
+  mkdir build
+  cd build
+
+  # CMake flags
+  export CMAKE_FLAGS="$CMAKE_FLAGS -DENABLE_TESTING=OFF"
```

Comment:
There is no need to export these, and using a more compact format would make things more readable if doing this in a script, i.e.
```
${CMAKE_EXE} -DUSE_SYSTEM_EIGEN:BOOL=ON \
  -DUSE_SYSTEM_GLEW:BOOL=ON \
  ..
make -j2
```

Reply:
The type specification is optional, but I use it in most scripts.